canonical/ubuntu.com
# Apply /desktop redirects

## Done

- Redirected /desktop/partners to c.com/partners/desktop and removed link from subnav
- Redirected /desktop/statistics to /desktop

## QA

- View the site locally in your web browser at: https://ubuntu-com-13488.demos.haus/desktop/statistics and see that you are rerouted to /desktop
- Visit https://ubuntu-com-13488.demos.haus/desktop/partners and see that you are rerouted to c.com/partners/desktop
- See that the partners link has been removed from the subnav

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8271 , https://warthogs.atlassian.net/browse/WD-8272
